op-chain-ops/genesis/setters.go

```diff
@@ -70,6 +77,7 @@ func setProxies(db vm.StateDB, proxyAdminAddr common.Address, namespace *big.Int
 		case predeploys.FiatTokenV2_2Addr:
 			db.SetCode(addr, fiatTokenProxyBytecode)
 			db.SetState(addr, AdminSlotForZepplin, eth.AddressAsLeftPaddedHash(proxyAdminAddr))
+		case predeploys.NonfungibleTokenPositionDescriptorAddr:
```

Comment:
Hi @AaronLee22 
Correct me if I am wrong, NonfungibleTokenPositionDescriptorAddr is not a proxy contract?

Reply:
Oh, you are right it was actually an implementation contract for transparentUpgradableProxy. let me fix this part! Thank you!